### PR TITLE
Add logos for buildah

### PR DESCRIPTION
**Pull request by rhatdan**
Thanks to Máirín Duffy for building these logos for thBuildah project

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by nalind** [comment]
@TomSweeneyRedHat Oh, I missed that.  The link should probably be [this](https://cdn.rawgit.com/projectatomic/buildah/master/logos/buildah.svg), but it won't be valid until this is merged.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-[buildah](https://www.youtube.com/embed/YVk5NgSiUw8) - a tool which facilitates building OCI container images
+![buildah logo](https://cdn.rawgit.com/projectatomic/buildah/logos/buildah.svg)
```
Comment (nalind) [inline README.md]: I think this needs a `master` between `buildah` and `logos` to point to a branch.

Comment (TomSweeneyRedHat) [reply README.md]: 👍

**Comment by rhatdan** [comment]
@nalind @TomSweeneyRedHat I fixed the link and fixed a few references to Buildah the project to be capitalized.  

I don't think you will see the logo until this is merged.

File: docs/buildah.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
-## buildah "1" "March 2017" "buildah"
+## Buildah "1" "March 2017" "buildah"
```
Comment (TomSweeneyRedHat) [inline docs/buildah.md]: nit, should the second buildah be capped?

**Comment by TomSweeneyRedHat** [comment]
Travis CI is failing, looks like the RPM build test isn't happy?  The closest thing to an error that I can see is:
#   Running scriptlet: glibc-2.25-10.fc26.x86_64                            93/93BDB1539 Build signature doesn't match environment
# go-compilers-golang-compiler-1-11.fc26.x86_64 was supposed to be installed but is not!

Can't imagine this submit had anything to do with that, maybe a one off?

**Comment by TomSweeneyRedHat** [comment]
Hmmm.  Same rpm building error in Red Hat CI.

**Comment by nalind** [comment]
bot, retest this please

**Comment by nalind** [comment]
@rh-atomic-bot, retest this please

**Comment by nalind** [comment]
@rh-atomic-bot, retest this please

**Comment by nalind** [comment]
bot, retest this please

**Comment by TomSweeneyRedHat** [comment]
Still the same error.  :^(

**Comment by rhatdan** [comment]
@cevich @jlebon  any ideas?

**Comment by cevich** [comment]
looking

**Comment by cevich** [comment]
@rhatdan here's the error:

```
make[1]: Leaving directory '/root/gopath/src/github.com/projectatomic/buildah/docs'
can't load package: package github.com/projectatomic/buildah/vendor/github.com/ostreedev/ostree-go/pkg/otadmin: 
vendor/github.com/ostreedev/ostree-go/pkg/otadmin/admin.go:1:1: expected 'package', found 'EOF'
can't load package: package github.com/projectatomic/buildah/vendor/github.com/ostreedev/ostree-go/pkg/otremote: 
vendor/github.com/ostreedev/ostree-go/pkg/otremote/remoteadd.go:1:1: expected 'package', found 'EOF'
WARNING: deadline exceeded by linter varcheck on . (try increasing --deadline)
WARNING: deadline exceeded by linter varcheck on cmd/buildah (try increasing --deadline)
WARNING: deadline exceeded by linter vet on . (try increasing --deadline)
WARNING: deadline exceeded by linter vet on cmd/buildah (try increasing --deadline)
make: *** [Makefile:50: validate] Error 2
make: Leaving directory '/root/gopath/src/github.com/projectatomic/buildah'
### EXITED WITH CODE 2 AFTER 449s
```

Best guess is some problem with github (happens on occasion).  Though I dunno Go well at all, so maybe a problem w/ the code?

**Comment by cevich** [comment]
@rhatdan Aye, looks like a problem with github.  Try going here (I get EOF as well):

https://raw.githubusercontent.com/projectatomic/buildah/master/vendor/github.com/ostreedev/ostree-go/pkg/otadmin/adminupgrade.go

**Comment by rhatdan** [comment]
@cgwalters Why do these files exists. They are all empty  should ostree-go drop them?

**Comment by cgwalters** [comment]
https://github.com/ostreedev/ostree-go/commit/cb6250d5a6a240b509609915842f763fd87b819d

**Comment by TomSweeneyRedHat** [comment]
bot, retest this please

**Comment by TomSweeneyRedHat** [comment]
@rh-atomic-bot, retest this please

**Comment by TomSweeneyRedHat** [comment]
Not having luck getting the test to kick off.  I'm gonna close and reopen this bad boy, that should do it.

**Comment by TomSweeneyRedHat** [comment]
Ok, tests kick off this time.

**Comment by TomSweeneyRedHat** [comment]
Yeah Green lights!  @rhatdan or @nalind, let's turn on the logo!

**Pull request merged**